actors/power/src/lib.rs

```diff
@@ -364,6 +367,32 @@ impl Actor {
         })
     }
 
+    /// Returns the total raw power of the network.
+    fn network_raw_power(rt: &mut impl Runtime) -> Result<NetworkRawPowerReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let st: State = rt.state()?;
+
+        Ok(NetworkRawPowerReturn { raw_byte_power: st.total_raw_byte_power })
```

Comment:
We also have a version of this frozen at the end of the last epoch. It's used by the built-in market actor in order to remove any advantage to re-ordering transactions. If I were to choose just one of the two, I think the "this epoch" version might be better to expose. I think it's reasonable to commit to this behaviour for the future, even if the built-in market stops using it or stops being built-in.

We could also expose both the stable version and instantaneous.

Reply:
I yo-yoed on this, and went with the unfrozen number because it was the more intuitive one. I didn't feel strongly, though, so I'm going to switch to the stable version on your suggestion.

I would be pretty opposed to exposing both, I worry that would only confuse.